.ocamlformat

```diff
@@ -0,0 +1,23 @@
+version = 0.21.0
```

Comment:
Feel free to change the .ocamlformat directly in this branch. I just copy/past the one from Deku for homogeneity.